docs/dev-guides/7-xrpackage-api.md

```diff
@@ -53,6 +53,10 @@ The `XRPackageEngine` API is documented [in the next section](./8-xrpackage-engi
 
 ## `getVolumeMesh`
 
+**Parameters**: None
+
+**Returns**: An <a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object" target="_blank" rel="noopener noreferrer">`Object`</a> containing metadata and geometry data for the package if a volume mesh exists, or `null` if a volume mesh does not exist.
```

Comment:
Could describe the object shape better.

Reply:
I added an example object but I wasn't sure what some of the fields were (Array of floating-point numbers/integers), do you think its fine to leave it like that or could you provide more context as to what those numbers are referring to? Thanks 😄

Reply:
It's actually a THREE.js scene I believe: https://github.com/webaverse/xrpackage/blob/master/xrpackage.js#L2068

We don't need to show the scene, but we should say what it actually is that's returned from the code.

Reply:
Oh awesome, will definitely fix that up haha